openedx/features/course_experience/templates/course_experience/course-updates-fragment.html

```diff
@@ -29,20 +29,26 @@
         </div>
     </header>
     <div class="page-content">
-        % if len(plain_html_updates) > 0:
+        % if plain_html_updates and len(plain_html_updates) > 0:
```

Comment:
I think this can be simply:

    % if plain_html_updates:

since an empty list is falsey.